### ISSUE TITLE
Get rid of jquery

### ISSUE THREAD

**Issue by asbiin** [monicahq/monica#3287]
We should not need jquery.

**Comment by djaiss** [comment]
I agree. We should remove most dependencies like that. Ideally, we would also reduce as much as possible the number of packages we have (both Vue and PHP), in order to reduce the number of point of failures in case a library is not maintained.
